### DISCUSSION TITLE
part-ii/2012-27/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#353] [Question comments]
Part II 2012 Coding and Cryptography: Paper 2, Section I, G 

https://questions.tripos.org/part-ii/2012-27/

**Comment by walrus1729** [comment]
<details>
<summary>Solution</summary>

This is all bookwork. The perfect linear code of length $n$ and minimum weight $3$ is the Hamming $(n,n-r)$-code. 

</details>
